docs/runbooks/LowReadyVirtOperatorsCount.md

```diff
@@ -0,0 +1,29 @@
+# LowReadyVirtOperatorsCount 
+
+## Meaning
+
+The virt-operator is the first k8s operator coming alive in a KubeVirt cluster. Its primary responsibilities are installation, live-update, live-upgrade of a KubeVirt cluster, monitoring the life-cycle of top-level controllers, such as virt-controller, virt-handler, virt-launcher, etc. and manage their reconciliation. In addition, virt-operator is responsible for cluster-wide tasks, such as certificate rotation and some infrastructure management, etc.
+
+Note that virt-operator is not directly responsible for virtual machines in the cluster, its temporal unavailability should not affect the custom workloads. 
+
+This alert is triggered when some virt operators are running but, not in the `Ready` state in the past 5 minutes. The virt-operator deployment has a default replica of 2 pods.
+
+## Impact
+
+In these circumstances the virt-operator will become a single point of failure.
```

Comment:
The rule of this alert is  `kubevirt_virt_operator_ready_total <  kubevirt_virt_operator_up_total`

This explanation is OK when kubevirt_virt_operator_ready_total==1 but what about kubevirt_virt_operator_ready_total == 0 ? When users observe this alert, they should check `NoReadyVirtOperator` as well. If `NoReadyVirtOperator` alert was not triggered, this explanation is OK. Otherwise, the problem is much critical than as stated here.